SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -396,11 +402,11 @@ SCDocHTMLRenderer {
                 stream << "<span class='soft'>" << this.escapeSpecialChars(node.text) << "</span>";
             },
             \ANCHOR, {
-                stream << "<a class='anchor' name='" << node.text << "'>&nbsp;</a>";
+				stream << "<a class='anchor' name='" << this.escapeSpacesInAnchor(node.text) << "'>&nbsp;</a>";
```

Comment:
The spacing is off here. Shouldn't introduce all that white space; it should be even with the other ones on the page.